src/ostorlab/serve_app/agent_groups/autodiscovery.yaml

```diff
@@ -1,6 +1,6 @@
 kind: AgentGroup
 description: Enumerate domain scan
-name: autodiscovery
+name: predefined_oxo_autodiscovery
```

Comment:
The `name` is required, right? The examples in https://oxo.ostorlab.co/tutorials/examples don't have `name`. So we should either update them 👍 or make `name` optional 👎

Reply:
as discussed this has nothing to do with optional.